src/nv_ingest/extraction_workflows/pdf/pdfium_helper.py

```diff
@@ -63,9 +64,21 @@ def extract_tables_and_charts_using_image_ensemble(
 ) -> List[Tuple[int, object]]:  # List[Tuple[int, CroppedImageWithContent]]
     tables_and_charts = []
 
+    # Obtain yolox_version
+    # Assuming that the grpc endpoint is at index 0
+    yolox_http_endpoint = config.yolox_endpoints[1]
+    try:
+        yolox_version = get_version(yolox_http_endpoint)
```

Comment:
Lets update this so that if 'get_version' fails, we assume the latest version. Its not ideal, but doing it like this assumes that the endpoint has to have an HTTP service available.

Reply:
Updated in cc30fc5.